src/components/ColorInputFlyout/ColorPickerFlyout.tsx

```diff
@@ -14,7 +14,9 @@ export type ColorPickerFlyoutProps = Pick<ColorPickerProps, "palettes" | "onSele
     onClose?: () => void;
     currentColor: Color | null;
     clearable?: boolean;
+    showDelete?: boolean;
```

Comment:
This is not needed because you can check if `onDelete` is set or not

Reply:
Removed showDelete

Reply:
Somehow it still shows up as a change? 🤔